index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+    <header class="navbar container fixed-top">
+        <a href="" id="wel">Welcome</a>
```

Comment:
- [ ] Kindly remove the text transform from your logo to match the figma design:
![image](https://user-images.githubusercontent.com/91486791/219732724-4f778426-fd17-4cbf-818c-5b130ef3093a.png)